keylime/tpm/tpm_main.py

```diff
@@ -1039,51 +1039,77 @@ def create_quote(
 
         return "r" + quote
 
-    def __tpm2_checkquote(
-        self, pubaik: str, nonce: str, quoteFile: str, sigFile: str, pcrFile: str, hash_alg: Union[Hash, str]
+    def tpm2_checkquote(
+            self,
+            pubaik: str,
+            nonce: str,
+            quote_file: str,
+            sig_file: str,
+            pcr_file: str,
+            hash_alg: Union[Hash, str],
     ) -> cmd_exec.RetDictType:
+        """Executes the tpm2_checkquote command.
+
+        Args:
+            pubaik: The public AIK file.
+            nonce: The nonce to use.
+            quote_file: The quote file.
+            sig_file: The signature file.
+            pcr_file: The PCR file.
+            hash_alg: The hash algorithm to use.
+
+        Returns:
+            A dictionary containing the stdout and stderr of the command.
+        """
         nonce = bytes(nonce, encoding="utf8").hex()
         if self.tools_version == "3.2":
             command = [
                 "tpm2_checkquote",
                 "-c",
                 pubaik,
                 "-m",
-                quoteFile,
+                quote_file,
                 "-s",
-                sigFile,
+                sig_file,
                 "-p",
-                pcrFile,
+                pcr_file,
                 "-G",
                 hash_alg,
                 "-q",
                 nonce,
             ]
         else:
-            # versions >= 4.0
             command = [
                 "tpm2_checkquote",
                 "-u",
                 pubaik,
                 "-m",
-                quoteFile,
+                quote_file,
                 "-s",
-                sigFile,
+                sig_file,
                 "-f",
-                pcrFile,
+                pcr_file,
                 "-g",
                 hash_alg,
                 "-q",
                 nonce,
             ]
 
-        retDict = self.__run(command, lock=False)
-        return retDict
+        ret_dict = cmd_exec.run(command, lock=False)
```

Comment:
that change it is not the same, the local `__run` should be indeed rewritten, but you are using parameters that make sense in `__run` but not in `run`, like `lock`

the rename of variables makes sense

Reply:
> the rename of variables makes sense

It only makes sense if we all agree that we don't want CamelCase but underscore everywhere from now on.

Reply:
The agreement is mostly implicit, as we are following pep8:

https://peps.python.org/pep-0008/#function-and-variable-names

```
mixedCase is allowed only in contexts where that’s already the prevailing style (e.g. threading.py), to retain backwards compatibility.
```